pkg/costmodel/cluster.go

```diff
@@ -851,6 +852,7 @@ func ClusterLoadBalancers(client prometheus.Client, start, end time.Time) (map[L
 
 			hrs := (lb.Minutes * scaleFactor) / 60.0
 			lb.Cost += lbPricePerHr * hrs
+			lb.Ip = providerID
```

Comment:
let's add another warning log here. If this lb object already has a non null IP, and that IP does not equal the provider ID, let's log a warning "warning - multiple IPs per load balancer not supported, picking most recent one" or something to that effect